gradle/libs.versions.toml

```diff
@@ -1,23 +1,23 @@
 [versions]
 
-kotlin = "1.8.10"
-gradle = "8.0.0"
+kotlin = "1.9.10"
+gradle = "8.2.0"
 gradle-nexus = "1.1.0"
-androidx-core = "1.9.0"
-androidx-test-rules = "1.4.0"
-camerax = "1.2.3"
-compose-bom = "2023.04.01"
-compose-compiler = "1.4.4"
-appcompat = "1.7.1"
-material = "1.6.1"
-lifecycle = "2.6.1"
-navigation = "2.5.2"
-accompanist = "0.25.1"
-coil = "2.2.2"
+androidx-core = "1.12.0"
+androidx-test-rules = "1.5.0"
+camerax = "1.3.1"
+compose-bom = "2023.10.01"
+compose-compiler = "1.5.6"
+appcompat = "1.8.1"
+material = "1.11.1"
```

Comment:
The [latest version](https://github.com/material-components/material-components-android/releases) is 1.10.0

Reply:
I see a 1.11.0 (not sure where I got the .1 from, updated): https://github.com/material-components/material-components-android/releases/tag/1.11.0